public/locations-list-map.js

```diff
@@ -254,234 +263,183 @@ window.onFilterChange = function (elem, scrollNeeded) {
   let states = null;
   document.filters['states'].forEach((state) => {
     if (state.checked) {
-      if (states === null) {
-        states = {};
-      }
+      states = states || {};
       states[state.value] = true;
     }
   });
 
-//  let acceptOpens = null;
-//  document.filters['accept-open'].forEach((acceptOpen) => {
-//    if (acceptOpen.checked) {
-//      if (acceptOpens === null) {
-//        acceptOpens = {};
-//      }
-//      acceptOpens[acceptOpen.value] = true;
-//    }
-//  });
-
   let acceptItems = null;
   document.filters['accept-item'].forEach((acceptItem) => {
     if (acceptItem.checked) {
-      if (acceptItems === null) {
-        acceptItems = {};
-      }
+      acceptItems = acceptItems || {};
       acceptItems[acceptItem.value] = true;
     }
   });
 
-  const filters = {states, acceptItems};
-  const htmlSnippets = toHtmlSnippets(window.data_by_location, filters);
-  const locationsListElement = document.getElementById('locations-list');
-  locationsListElement.innerHTML = htmlSnippets.join(" ");
+  const filters = { states, acceptItems };
+  const locationsList = $(".locations-list");
+
+  locationsList.empty().append(getFilteredContent(data_by_location, filters));
+  showMarkers(data_by_location, filters, false);
+
   if (scrollNeeded) {
-    locationsListElement.scrollIntoView({'behavior': 'smooth'});
+//    locationsList[0].scrollIntoView({'behavior': 'smooth'});
```

Comment:
is this intentionally commented out?

Reply:
Oops. I did that for testing, since it was seriously annoying while working on the filter updates. I'll put it back to preserve existing behavior, but do think this should be revisited.